Update known-issues.md (#1678)
maticnetwork/matic-docs: docs/maintain/validate/kb/known-issues.md

### Node is not signing any checkpoints

> **PreRequisite:**
> On the first point please point the bor_rpc_url(heimdall-config.toml) of the validator to any external infra providers and restart the services.This change helps to avoid missing checkpoints.
> 
> Our docs has shared a list of Infra Providers which validator can make use of.
> 
> :::note
> 
> At this point of time the node will not mine blocks.So once the issue is fixed the changes made has to be reverted for the node to return to the normal functionality.
> 
> :::
> 

Reviewer: ```suggestion
**PreRequisite:**
First, please point the bor_rpc_url(heimdall-config.toml) of the validator to any external infra providers and restart the services. This change helps to avoid missing checkpoints.

You can find a list of Infra Providers, on the navbar, that validators can make use of.

:::note

At this point in time, the node will not mine blocks. So once the issue is fixed, the changes made have to be reverted for the node to return to normal functionality.

:::

```
